erpnext/stock/doctype/repost_item_valuation/repost_item_valuation.py

```diff
@@ -281,6 +281,7 @@ def repost(doc):
 		repost_gl_entries(doc)
 
 		doc.set_status("Completed")
+		remove_attached_file(doc.name)
```

Comment:
@rohitwaghchaure we can also do this in `clear_logs` function, recent files can be useful for debugging someday?

Reply:
Better we remove at this time only